tools/ionizer/options.c

```diff
@@ -10,8 +10,8 @@
 #include <stdio.h>
 #ifdef WIN
 #include <io.h>
-#endif
 #include <malloc.h>
+#endif
```

Comment:
I left it in based on the [MSDN `malloc` docs](https://msdn.microsoft.com/en-us/library/6ewkz86d.aspx), but I have no way to test it.